.github/workflows/deploy.yml

```diff
@@ -24,6 +24,6 @@ jobs:
       - name: Deploy
         uses: JamesIves/github-pages-deploy-action@releases/v3
         with:
-          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          GITHUB_TOKEN: ${{ github.token }}
```

Comment:
These two values should be identical. Newer documentation I saw tended to prefer `github.token`.